debian/control.in

```diff
@@ -7,6 +7,16 @@ Build-Depends: debhelper (>= 9), autotools-dev, flex, postgresql-server-dev-all
 Standards-Version: 3.9.7
 Homepage: https://github.com/citusdata/citus-ha
 
+Package: citus-ha
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Provides: citus-ha
+Conflicts: citus-ha
+Recommends: postgresql-10-citus-ha | postgresql-11-citus-ha
+Suggests: postgresql-11-citus-8.1 | postgresql-10-citus-8.1 | postgresql-10-citus-enterprise-8.1 | postgresql-11-citus-enterprise-8.1
+Description: Citus high availability CLI
+ This CLI is used to manage HA installations of a Citus cluster.
+
 Package: postgresql-PGVERSION-citus-ha
```

Comment:
add requirement to citus-ha

Reply:
added `citus-ha` to `Depends:`, verified in docker that it depends on it and is installable together with that package